package.json

```diff
@@ -62,14 +62,9 @@
     "eslint": "8.17.0",
     "eslint-config-standard": "17.0.0",
     "eslint-config-standard-jsx": "11.0.0",
-    "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jest": "26.5.3",
     "eslint-plugin-jsx-a11y": "6.5.1",
-    "eslint-plugin-n": "15.2.2",
```

Comment:
same here: do we want to have this? if yes we should activate it instead of removing